sdk/core/azure-core-tracing-opentelemetry/setup.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import re
+import os.path
+from io import open
+from setuptools import find_packages, setup  # type: ignore
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-core-tracing-opentelemetry"
+PACKAGE_PPRINT_NAME = "Azure Core OpenTelemetry plugin"
+
+package_folder_path = "azure/core/tracing/ext/opentelemetry_span"
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, '__init__.py'), 'r') as fd:
+    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',  # type: ignore
+                        fd.read(), re.MULTILINE).group(1)
+
+if not version:
+    raise RuntimeError('Cannot find version information')
+
+with open('README.md', encoding='utf-8') as f:
+    readme = f.read()
+with open('HISTORY.md', encoding='utf-8') as f:
+    history = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description='Microsoft Azure {} Library for Python'.format(PACKAGE_PPRINT_NAME),
+    long_description=readme + '\n\n' + history,
+    long_description_content_type='text/markdown',
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email='azpysdkhelp@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/core/azure-core-tracing-opentelemetry',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'License :: OSI Approved :: MIT License',
+    ],
+    zip_safe=False,
+    packages=[
+        'azure.core.tracing.ext.opentelemetry_span',
+    ],
+    install_requires=[
+        'opentelemetry-api>=0.2a0',
+        'opentelemetry-ext-azure-monitor>=0.2a0',
```

Comment:
should not be required

Reply:
We discussed that with @johanste , and yes we know it's not required, but we though it would be nice to install it at the same time? Like advertising the product :) Thoughts?

Reply:
installing it is not enough anyway (someone needs to configure it and follow Azure Monitor guidance, create a resource, etc). 
So, I'd rather not require it and tackle this problem from the Azure Monitor side: we should make integration for python apps (perhaps hosted in Azure) easy. E.g. for .NET AppService, it's 1 button click and eventually, we should get to a similar experience with python.